pattern-library/sass/_normalize.scss

```diff
@@ -355,24 +355,12 @@ input[type="number"]::-webkit-outer-spin-button {
  */
 
 input[type="search"] {
-  -webkit-appearance: textfield; /* 1 */
   -moz-box-sizing: content-box;
   -webkit-box-sizing: content-box; /* 2 */
   box-sizing: content-box;
 }
 
 /**
- * Remove inner padding and search cancel button in Safari and Chrome on OS X.
```

Comment:
That's a reasonable theory. I think the way you have it is better, because folks who are used to Chrome or Safari would expect to see the cancel button. I can see that we might want to implement the same behavior into Firefox and IE at some point, but it doesn't seem very high priority.